Documentation/ApiOverview/BackendModules/ModuleConfiguration.rst

```diff
@@ -0,0 +1,287 @@
+   if not set.
+
+.. confval:: access
+
+   :Scope: Backend module configuration
+   :type: string
+
+   Can be `user` (editor permissions), `admin`, or  `systemMaintainer`.
+
+.. confval:: workspaces
+
+   :Scope: Backend module configuration
+   :type: string
+
+   Can be `*` (= always), `live` or `offline`
+
+
+.. confval:: position
+
+   :Scope: Backend module configuration
+   :type: array
+
+   The module position. Allowed values are `top` and `bottom` as
+   well as the key value pairs `before => <identifier>` and
+   `after => <identifier>`.
+
+.. confval:: appearance
+
+   :Scope: Backend module configuration
+   :type: array
+
+   Allows to define additional appearance options. Currently only
+   :confval:`appearance.renderInModuleMenu` is available.
+
+.. confval:: appearance.renderInModuleMenu
+
+   :Scope: Backend module configuration
+   :type: bool
+
+   If set to false the module is not displayed in the module menu.
+
+.. confval:: iconIdentifier
+
+   :Scope: Backend module configuration
+   :type: string
+
+   The module icon identifier
+
+.. confval:: icon
+
+   :Scope: Backend module configuration
+   :type: string
+
+   Path to a module icon (Deprecated: Use :confval:`iconIdentifier` instead)
+
+
+.. confval:: labels
+
+   :Scope: Backend module configuration
+   :type: array or string
+
+   An :php:`array` with the following keys:
+
+   -  `title`
+   -  `description`
+   -  `shortDescription`
+
+   The value can either be a static string or a locallang label reference.                                                       |
+
+   It is also possible to define the path to a locallang file.
+   The referenced file should contain the following label keys:
+
+   -  `mlang_tabs_tab` (Used as module title)
+   -  `mlang_labels_tabdescr` (Used as module description)
+   -  `mlang_labels_tablabel` (Used as module short description)
+
+
+.. confval:: component
+
+   :Scope: Backend module configuration
+   :type: string
+   :Default: TYPO3/CMS/Backend/Module/Iframe
+
+   The view component, responsible for rendering the module.
+
+
+.. confval:: navigationComponent
+
+   :Scope: Backend module configuration
+   :type: string
+
+   The module navigation component. The following are provided by the Core:
+
+   `TYPO3/CMS/Backend/PageTree/PageTreeElement`
+      The page tree as used in the Web module.
+
+   `TYPO3/CMS/Backend/Tree/FileStorageTreeContainer`
+      The file tree as used in the Filelist module.
+
+
+.. confval:: navigationComponentId
+
+   :Scope: Backend module configuration
+   :type: string
+
+   The module navigation component (Deprecated: Use
+   :confval:`navigationComponent`)
+
+
+.. confval:: inheritNavigationComponentFromMainModule
+
+   :Scope: Backend module configuration
+   :type: bool
+   :Default: true
+
+   Whether the module should use the parents navigation component.
+   This option defaults to :php:`true` and can therefore be used to
+   stop the inheritance for sub modules.
+
```

Comment:
.. confval:: moduleData

The allowed module data properties and their default value.  Module data are the module specific settings of a backend user. The properties, defined in the registration, can be overwritten in a request via :php:`GET` or :php:`POST`. For more information about the usage of this option, see the corresponding :doc:`changelog <Feature-96895-IntroduceModuleDataObject>`.